_posts/2018-02-28-swiftfall.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title:  Swiftfall
+date:   2018-02-228 21:25:00
+categories: projects
+description: A wrapper for an API called Scryfall.
+author: Braden Bowdish
+author-image: https://avatars2.githubusercontent.com/u/15185628?s=460&v=4
+author-bio: Lover of Swift and Magic. 3rd year member of  Computer Science House
+author-email: bmbowdish@csh.rit.edu
+author-social:
+  github: https://github.com/bmbowdish
+---
+
+# What is Swiftfall? 
+
+Swiftfall is a wrapper written in Swift for the API Scryfall.
+
+[Documentation for Scryfall API.](https://scryfall.com/docs/api)
+
+Scryfall is API which handles information about the card game Magic: The Gathering. 
+
+I like Magic the Gathering, but getting tons of information about Magic cards is really annoying. Especially if you don't want to or don't know how to implement a JSON parser or making requests to a website. Many newer developers would like to combine their passions but it's not always easy. 
+
+Thankfully, someone already found a solution to this problem. [Scrython does this already, but in Python.](https://github.com/nandascott/Scrython) Despite this, I decided to work on a similar project in Swift. 
+
+### Why Swift Was Appealing
+1) Swift's use of optionals was really appealing.
+2) Swift has a JSON decoder built in and it is appealing to have no dependencies.
+3) iOS, Apple TV, Apple Watch, and MacOS development is easy in Swift
+4) Swift is new, and it is fun to do things never done before 
```

Comment:
Clarify `do things never done before`. Are you excited to learn Swift? Are you excited to write a library in a language that is missing such a library?